lib/riemann/version.rb

```diff
@@ -1,3 +1,3 @@
 module Riemann
-  VERSION = '0.2.6'
+  VERSION = '0.3.0'
```

Comment:
I bumped the minor version but we still are in the 0.x era. I am wondering if it makes sense to start saying that this code is ready for production use and bump to 1.0.0?